src/gobexport/export.py

```diff
@@ -213,13 +215,12 @@ def cleanup_datefiles(connection, container, filename):
         return
 
     logger.info(f'Clean previous files for {filename}.')
-    try:
+
+    with suppress(Exception):
```

Comment:
Can we remove this?

I know you're not changing the logic, and that the exception was suppressed already, but I'd rather not see any suppressing of Exceptions. If we need to suppress a certain Exception, we should catch only that Exception (instead of all Exceptions).

Can you try what happens when you remove the except clause, and change it so that we only ignore the minimum Exception necessary? Suppressing all exceptions may cause a lot of trouble down the line.

Reply:
I changed it to suppressing Type and KeyError. Is this narrow enough? I can't see atm which specific errors maybe raised down the line..

Reply:
Why would we expect a TypeError and KeyError? I'd expect that if one of those exceptions are raised it is because something really went wrong.

My hunch is that delete_object may raise an exception when a file does not exist, and that that was the reason if ignoring all exceptions.

What you can do too; just remove the exception handler altogether, and see what starts failing :)

Reply:
Thx, removed it